src/axom/mint/docs/sphinx/sections/fem.rst

```diff
@@ -0,0 +1,253 @@
+.. ## Copyright (c) 2017-2019, Lawrence Livermore National Security, LLC and
+.. ## other Axom Project Developers. See the top-level COPYRIGHT file for details.
+.. ##
+.. ## SPDX-License-Identifier: (BSD-3-Clause)
+
+
+.. _sections/fem:
+
+Finite Elements
+-----------------
+
+The Finite Element Method (FEM) is a widely used numerical technique, employed
+for the solution of *partial differential equations* (PDEs), arising in
+*structural engineering analysis* and more broadly in the field of
+*continuum mechanics*.
+
+However, due to their generality and mathematically sound foundation,
+:ref:`sections/fem`, are often employed in the implementation of other numerical
+schemes and for various computational operators, e.g., interpolation,
+integration, etc.
+
+Mint provides basic support for :ref:`sections/fem` that consists:
+
+#. :ref:`LagrangeBasis` *shape functions* for commonly employed :ref:`CellTypes`
+#. Corresponding :ref:`Quadratures` (under development)
+#. Routines for forward/inverse :ref:`IsoparametricMapping`, and
+#. Infrastructure to facilitate adding *shape functions* for
+   new :ref:`CellTypes`, as well as, to :ref:`AddABasis`.
+
+This functionality is collectively exposed to the application through the
+``mint::FiniteElement`` class. Concrete examples illustrating the usage of the
+``mint::FiniteElement`` class within an application code are provided in the
+:ref:`femTutorial` tutorial section.
+
+A *Finite Element Basis* consists of a family of *shape functions*
+corresponding to different :ref:`CellTypes`. Mint currently supports
+Lagrange isoparametric :ref:`sections/fem`.
+
```

Comment:
Add an additional mention (somewhere in here) to https://cgns.github.io/CGNS_docs_current/sids/conv.html#unstructgrid, since node order is intimately connected with correct FEM computation.

Reply:
Done.